setup.cfg

```diff
@@ -49,7 +49,10 @@ ignore = E203,  # space before : (needed for how black formats slicing)
          E711,  # comparison to none should be 'if cond is none:'
 
 exclude =
-    doc/source/development/contributing_docstring.rst
+    doc/source/development/contributing_docstring.rst,
+    # work around issue of undefined variable warnings
+    # https://github.com/pandas-dev/pandas/pull/38837#issuecomment-752884156
+    doc/source/getting_started/comparison/includes/*.rst
```

Comment:
Per suggestion in https://github.com/pandas-dev/pandas/pull/38837#issuecomment-752884156.